addons/html_editor/static/tests/_helpers/plugins.js

```diff
@@ -0,0 +1,20 @@
+import { Plugin } from "@html_editor/plugin";
+import { parseHTML } from "@html_editor/utils/html";
+
+export class SimplePlugin extends Plugin {
```

Comment:
Could you add this test plugin directly in the test function ? If we see that we are using it a lot in other tests we'll make a test util like you did, but I don't think it's necessary right now